spec_files/mesa/mesa.spec

```diff
@@ -73,7 +73,7 @@ Summary:        Mesa graphics libraries
 Version:        %{lua:ver = string.gsub(rpm.expand("%{ver}"), "-", "~"); print(ver)}
 Epoch:          1
 %global orig_release 1%{?dist}
-Release:        100.bazzite.%{orig_release}
+Release:        101.bazzite.%{orig_release}
```

Comment:
not sure if changing this is needed, but then again rpm packaging is not my forte, but i think the release is generated from orig_release.